packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1450,6 +1494,72 @@ class ImportedSubnet extends Resource implements ISubnet, IPublicSubnet, IPrivat
   }
 }
 
+export type RouteTargetType = "egressOnlyInternetGatewayId" | "gatewayId" | "instanceId"| "natGatewayId" |"networkInterfaceId" | "vpcPeeringConnectionId" | "transitGatewayId";
```

Comment:
This needs to be an enum. jsii does not support exporting union types (nor does it support types consisting of string literals).